templatescompiler/erbrenderer/template_evaluation_context.rb

```diff
@@ -169,22 +166,25 @@ def dump(*args)
 end
 
 class ERBRenderer
-  def initialize(context)
-    @context = context
+  def initialize(json_context_path)
+    @json_context_path = json_context_path
   end
 
   def render(src_path, dst_path)
-    erb = ERB.new(File.read(src_path), safe_level = nil, trim_mode = "-")
```

Comment:
minor nit: while i dont think i feel strongly about this (nor do i think it matters in this context since its already out of process from the cli), it is nice to explicitly call it out its running in-process